src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    {person.isMarried === true && person.sex === 'm' && (
```

Comment:
Avoid using strict equality and instead use truthy check when checking for 'true' or 'false'. This is because sometimes values might be truthy or falsy but not strictly equal to true or false.
```suggestion
    {person.isMarried && person.sex === 'm' && (
```